dbc-gateway.codegen.yml

```diff
@@ -5,7 +5,7 @@ schema:
         Authorization: "Bearer ${STORYBOOK_LIBRARY_TOKEN}"
 documents: "src/**/*.graphql"
 generates:
-  src/core/dbc-gateway/generated/graphql.tsx:
+  src/core/dbc-gateway/generated/graphql.ts:
```

Comment:
I am not sure it was a typo.
I think it was because the generated file contains hooks.
But if it doesn't matter that a hook-containing file is called `.ts` then so be it.

Reply:
According to the React docs, hooks should use the `.js` or `.ts` file extension
https://react.dev/learn/reusing-logic-with-custom-hooks